### ISSUE TITLE
Focus visibility and navigation (keyboard only)

### ISSUE THREAD

**Issue by NJMannai** [dnet/androsphinx#17]
Device: 

- Pixel 4a, Android 12, TalkBack 13. 
- Wireless keyboard. 

Navigation through onboarding with only a keyboard, the focus is not visible.
The search function and settings menu button can't be activated.

Once you're in the settings menu (through double tap) the focus visibility and activation with the keyboard is working. 
However, once you're in the search menu (after adding a URL), the keyboard navigation is stuck again. There is no focus visibility.

Related to WCAG 2.4.7 Focus visible and 2.1.1 Keyboard
